app/src/main/res/values/strings.xml

```diff
@@ -286,7 +286,7 @@
     <!--Use ← for LTR languages-->
     <string name="magic_keyboard_activate_setting_path_2_text">or (\"Settings\" → \"Language &amp; input\" → \"Virtual keyboard\" and pick one.)</string>
     <string name="keyboards_choose_magikeyboard_text">Choose the Magikeyboard when you need to fill a form.</string>
-    <string name="keyboards_swicth_magikeyboard_text">You can easily switch from your main keyboard to Magikeyboard with the language button of your keyboard, a long press on the space bar of your keyboard, or, if it is not available, with : </string>
+    <string name="keyboards_swicth_magikeyboard_text">Switch keybaords by long pressing the spacebar of your keyboard, or, if it is not available, with \":\" </string>
```

Comment:
You remove the information of "langage button", and ":" is not a button (you can remove it), it's just to show the image below the description.